### PR TITLE
Add initial socket ack to prevent false "running previous commands" message

### PR DESCRIPTION
**Pull request by thieman**
@jsingle

### CODE REVIEW

File: dusty/cli/__init__.py [hunk 1]
```diff
@@ -82,7 +82,9 @@ def _run_command(sock, command):
         if data:
             stripped = data.decode('utf-8').replace(constants.SOCKET_TERMINATOR, '').replace(constants.SOCKET_ERROR_TERMINATOR, '')
             sys.stdout.write(stripped)
-            if data.endswith(constants.SOCKET_TERMINATOR):
+            if stripped == constants.SOCKET_ACK:
```
Comment (jsingle) [inline dusty/cli/__init__.py]: do we not want `startswith` here?

**Comment by jsingle** [comment]
Looks good if you're sure we don't want `startswith` instead of ==, and it works in practice.  This is great, I  hate this bug

File: dusty/cli/__init__.py (hunk 1 above)
Comment (thieman) [inline dusty/cli/__init__.py]: Should be equivalent, I think. The entire send is just the socket ack.

Comment (jsingle) [inline dusty/cli/__init__.py]: So `recv` reads only 1 `sendall`'s worth at once?

Comment (thieman) [inline dusty/cli/__init__.py]: I think the complete answer there is a little complicated but with this tiny data size I am seeing that that's how it's working in practice

**Comment by thieman** [comment]
Alright actually I did end up needing to strip it out, it gets grouped with other messages sometimes

**Pull request merged**